### ISSUE TITLE
Next.js styled-components & emotion examples do not work using Yarn 2

### ISSUE THREAD

**Issue by Interpause** [ben-rogerson/twin.examples#21]
When using Yarn 2 Plug'n'Play and following the instructions exactly, for styled-components I get this error on running `next dev`:
```
error - ./.yarn/$$virtual/next-virtual-5d5df27552/0/cache/next-npm-10.0.4-e9908bac70-5ec6818d46.zip/node_modules/next/dist/client/next-dev.js
Error: Cannot find module '@emotion/babel-plugin' from 'e:\Documents\GitHub\repo'
    at Array.map (<anonymous>)
    at Generator.next (<anonymous>)
    at loadFileChain.next (<anonymous>)
```
As for emotion:
```
error - ./.yarn/$$virtual/next-virtual-5d5df27552/0/cache/next-npm-10.0.4-e9908bac70-5ec6818d46.zip/node_modules/next/dist/client/next-dev.js
Error: Cannot find module 'babel-plugin-macros' from 'e:\Documents\GitHub\repo'
    at Array.map (<anonymous>)
    at Generator.next (<anonymous>)
    at loadFileChain.next (<anonymous>)
```
I think it is likely an issue with Babel or Nextjs itself but I do not have the expertise to debug it. Works fine if I switch Yarn 2's nodeLinker to node-modules mode. So I am guessing likely a bad import somewhere.

**Comment by ben-rogerson** [comment]
Thanks for posting this issue, I'll take a look into it and get back to you.

**Comment by Interpause** [comment]
I was researching a bit and found that Yarn 2 has an option to set its PnP resolving mode to "loose" rather than the default of "strict" by adding pnpMode:"loose" to the .yarnrc. Doing so, and adding @emotion/babel-plugin and babel-plugin-macros to the dependencies was able to get it work without node_modules.

**Comment by ben-rogerson** [comment]
Nice, thanks for those tips.
I think a new docs page could show others the setup for yarn 2 (https://github.com/ben-rogerson/twin.macro/tree/master/docs/yarn2-installation.md).

Would you like to contribute? We could add a link to your repo/template at the bottom.

**Comment by Interpause** [comment]
cool, ill try doing it

**Comment by Interpause** [comment]
Hm, actually, while I could reproduce the issue just now, adding @emotion/babel-plugin and babel-plugin-macros was sufficient to fix it for emotionjs without having to change pnpMode. Maybe something was fixed since the time I ran into the issue? I guess it wouldn't be necessary to have a separate setup docs then. Which would mean that perhaps the examples should just be checked for missing dependencies.
